src/js/utils/dom.js

```diff
@@ -329,7 +328,7 @@ export function findElPosition(el) {
 
   // Android sometimes returns slightly off decimal values, so need to round
   return {
-    left: roundFloat(left),
```

Comment:
Ah, I see. Instead of `toFixed` with no arguments, use `Math.round`. Makes sense.